htdocs/talkpost_do.bml

```diff
@@ -176,6 +176,19 @@ body<=
     return LJ::bad_input($ML{'/talkpost.bml.error.noreply_readonly_journal'})
         if $journalu && $journalu->is_readonly;
 
+    # is the current user banned by the author of the original post?
+    return LJ::bad_input( $ML{'.error.banned.entryowner'} )
+        if ( defined $remote && $entryu->has_banned( $remote ) );  
+
+    # don't allow user A to reply to a comment by user B if user B has banned user A
+    if ( defined $remote ) {    #skip this if not logged in
+        my $parentu = LJ::load_userid( $parent->{posterid} );
+        return LJ::bad_input( $ML{'.error.banned.reply'} )
+            if ( defined $parentu && $parentu->has_banned( $remote ) );
+            #we check whether parentu is defined because it won't be
```

Comment:
For this comment, please just put the words in the top of the block and use capitalization and spacing. I.e., since this entire block is just doing a single check, it should all be in one comment.

```
# Comment comment
if ( ... ) {
    my $parentu = ...
    return ...
}
```

The idea here is that it means you can read through the comment first, know what the entire block of code does, and then look at the code. (This is because the block is so short, we don't want to spread comment text around as that makes your brain have to switch back and forth. Also, since the code is so short, it is basically self-documenting and if you do need to call something out it should just be on top.)